src/ui-services/notificationsService.js

```diff
@@ -18,6 +18,9 @@ angular.module('openshiftCommonUI').provider('NotificationsService', function()
     };
 
     var addNotification = function (notification) {
+      notification.id = notification.id || _.uniqueId('notification-');
+      // This format matches timestamp of API objects
+      notification.timestamp = moment.parseZone(new Date()).utc().format();
```

Comment:
I think just

```js
new Date().toISOString();
```

does what we want, then we don't need to add moment.js to common

Reply:
And it's simpler :)

Reply:
True. lets do that.